Logtail.NLog/LogtailTarget.cs

```diff
@@ -45,24 +43,54 @@ public sealed class LogtailTarget : TargetWithContext, IUsesStackTrace
         /// We capture the file and line of every log message by default. You can turn this
         /// option off if it has negative impact on the performance of your application.
         /// </summary>
-        public bool CaptureSourceLocation { get; set; } = true;
+        public bool CaptureSourceLocation
+        {
+            get => StackTraceUsage == StackTraceUsage.Max;
+            set => StackTraceUsage = value ? StackTraceUsage.Max : StackTraceUsage.WithoutSource;
```

Comment:
Looks like `StackTraceUsage.WithoutSource` is deprecate in NLog 5

> 'StackTraceUsage.WithoutSource' is obsolete: 'Replace with `WithStackTrace`. Marked obsolete on NLog 5.0'

```suggestion
            set => StackTraceUsage = value ? StackTraceUsage.Max : StackTraceUsage.WithStackTrace;
```

Reply:
I just realized that if we replace this it won't be compatible with NLog 4, makes sense to keep it in for now then.